content/docs/2.0/scalers/azure-log-analytics.md

```diff
@@ -0,0 +1,150 @@
+| summarize MetricValue=round(avg(CounterValue)) by CounterName, AppName
+| join (Perf 
+        | where InstanceName contains AppName
+        | where InstanceName contains ClusterName
+        | where CounterName == "cpuLimitNanoCores"
+        | where TimeGenerated > AvgDuration
+        | extend AppName = substring(InstanceName, indexof((InstanceName), "/", 0, -1, 10) + 1)
+        | summarize arg_max(TimeGenerated, *) by AppName, CounterName
+        | project Limit = CounterValue, TimeGenerated, CounterPath, AppName)
+        on AppName
+| project MetricValue, Threshold = Limit * ThresholdCoefficient
+```
+
+You can convert your query in JSON escaped string using [this](https://www.freeformatter.com/json-escape.html) tool and check yaml specific escapes.
+
+**Scaler limitations (READ CAREFULLY):**
+
+- As Log Analytics Scaler using Service Principal to authorize REST API requests, you should be aware of throttling. You should take to account the following AAD limit: 200 requests per 30 seconds. Each Log Analytics Scaler request's Access Token twice per pooling interval (one by Keda Operator and one by Keda Metrics Server). So, theoretical maximum number of Log Analytics scalers, created with the same Service Principal for 30sec pooling interval can be 100. Read more about throttling: [here](https://dev.applicationinsights.io/documentation/Authorization/Rate-limits), [here](https://docs.microsoft.com/en-us/previous-versions/azure/ad/graph/howto/azure-ad-graph-api-throttling#what-is-throttling) and [here](https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/directory-service-limits-restrictions)
+- As it was mentioned before, you can define a threshold using query (2d cell of query result will be interpret as threshold). Be aware! Threshold from query result will be set only once, during scaler creation. So, if your query will return different threshold values during runtime, they will not be propagated to HPA target.
+  
+### Authentication Parameters
+
+ You can use `TriggerAuthentication` CRD to configure the `tenantID`, `clientID`, `clientSecret` and `workspaceID` to connect to Log Analytics API.
+
+**Service Principal based authentication:**
+
+- `tenantID` required: Azure Active Directory TenantId. Follow [this](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-show) link to retrieve your TenantId.
+- `clientID` required: appId from your Service Principal. Follow [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) link to create your Service Principal.
+- `clientSecret` required: password from your Service Principal.
+- `workspaceID` required: Your Log Analytics WorkspaceId. Follow [this](https://docs.microsoft.com/en-us/cli/azure/monitor/log-analytics/workspace?view=azure-cli-latest#az-monitor-log-analytics-workspace-list) link to get your Log Analytics WorkspaceId.
+
+### Example

```

Comment:
In this case, it might be useful to include a screenshot of Kusto query result in Azure portal, but maybe that's a bit too far?

Reply:
Added screenshot in last commit, but in "### Query Guidance" section.

Reply:
Thanks!